other-markdown.Rmd

```diff
@@ -2,29 +2,37 @@
 
 ```{r, echo = FALSE}
 source("common.R")
-status("drafting")
+status("polishing")
 ```
 
 ## Introduction
 
-You now have a package that's ready to submit to CRAN.
-But before you do, there are two important files that you should update: `README.md` which describes what the package does, and `NEWS.md` which describes what's changed since the previous version.
-I recommend using Markdown for these files, because it's useful for them to be readable as both plain text (e.g. in emails) and HTML (e.g. on GitHub, in blog posts).
-The basic writing and formatting syntax is available at <https://help.github.com/articles/basic-writing-and-formatting-syntax/>.
+In this chapter we highlight two files that are conventionally used to provide some package-level documentation.
+These two are important, because they are featured on both the CRAN landing page and the pkgdown site for a package:
```

Comment:
And GitHub? 

I feel like we need a little something saying these are useful even if not submitting to CRAN?